### DISCUSSION TITLE
Search in custom fields

### DISCUSSION THREAD

**Discussion by presedo** [Shelf-nu/shelf.nu#1284] [Ideas]
It would be very useful to be able to filter the asset list by also searching in the custom fields.
For example, searching an asset by serial number

**Comment by DonKoko** [comment]
Hey @presedo . This is very much in progress. We are building an advanced view of the asset index that will allow you to filter by any custom field as well as many more things that are not available now. 
There is no PR yet, but here is the issue I am using to track the progress as well as note some stuff: https://github.com/orgs/Shelf-nu/projects/2?pane=issue&itemId=80390615&issue=Shelf-nu%7Cshelf.nu%7C1322
